test.sh

```diff
@@ -119,6 +119,7 @@ curl --url "pop3s://localhost:$POP_PORT" \
 
 CR=$(printf "\r")
 # Check that the user can send a message to the sever
+(
```

Comment:
what is the purpose of these parens? Could they be avoided by moving the heredoc to the very end? not sure how I feel about it either way.

Reply:
yeah parens create a subshell that the heredoc is passed in as stdin. I looked at a couple of ways to do it and I think this was the cleanest